File: app/src/test/java/com/tasktree/PositionTest.java
package com.tasktree;

import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class PositionTest {

    @Test
    public void givenTwoEqualPositionsThenEqualsReturnsTrue( ) {

        Position pos1 = new Position( 1 );
        Position pos2 = new Position( 1 );

        assertTrue( pos1.equals( pos1 ) );
        assertTrue( pos1.equals( pos2 ) );
        assertTrue( pos2.equals( pos1 ) );
    }

}

File: app/src/test/kotlin/com/tasktree/PositionTest.kt
package com.tasktree

import org.junit.Test

import org.junit.Assert.assertTrue

class PositionTest {

    @Test
    fun givenTwoEqualPositionsThenEqualsReturnsTrue() {

        val pos1 = Position(1)
        val pos2 = Position(1)

        assertTrue(pos1 == pos1)
        assertTrue(pos1 == pos2)
        assertTrue(pos2 == pos1)
    }

}